enterprise/server/hostedrunner/hostedrunner.go

```diff
@@ -174,6 +174,7 @@ func (r *runnerService) createAction(ctx context.Context, req *rnpb.RunRequest,
 			// Run from the scratch disk, since the workspace disk is hot-swapped
 			// between runs, which may not be very Bazel-friendly.
 			{Name: "WORKDIR_OVERRIDE", Value: "/root/workspace"},
+			{Name: "GIT_BRANCH", Value: req.GetRepoState().GetBranch()},
```

Comment:
In a later PR it would be great if we could also set `GIT_REPO_DEFAULT_BRANCH` by looking up the repo metadata using the GitHub app if it's installed. Otherwise I think we'll have a cold run on every new branch.

Reply:
Sounds good - will add that to my list